### ISSUE TITLE
Vue js error for staff profiles

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#1196]
Seen on multiple unit staff profiles pages, including Mann (related to bug #1195?) and Management (related to bug #1191?):

`DOMException: Failed to execute 'querySelector' on 'Document': The provided selector is empty.
    at https://johnson.library.cornell.edu/wp-content/themes/culu/vue/dist/js/staff.js?ver=6.2:1:22811
    at l (https://johnson.library.cornell.edu/wp/wp-includes/js/dist/vendor/regenerator-runtime.min.js?ver=0.13.11:1:1755)
    at o.value (https://johnson.library.cornell.edu/wp/wp-includes/js/dist/vendor/regenerator-runtime.min.js?ver=0.13.11:1:1568)
    at Generator.next (https://johnson.library.cornell.edu/wp/wp-includes/js/dist/vendor/regenerator-runtime.min.js?ver=0.13.11:1:2182)
    at r (https://johnson.library.cornell.edu/wp-content/themes/culu/vue/dist/js/chunk-vendors.js?ver=6.2:1:9123)
    at c (https://johnson.library.cornell.edu/wp-content/themes/culu/vue/dist/js/chunk-vendors.js?ver=6.2:1:9326)`

**Comment by jgreidy** [comment]
This seems to have gone away - replaced by another problem:

`jQuery.Deferred exception: e.dispatchEvent is not a function dispatch@https://johnson.library.cornell.edu/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.13.2:2:34589
init@https://johnson.library.cornell.edu/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.13.2:2:9028
5654/<@https://johnson.library.cornell.edu/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.13.2:2:9363
e@https://johnson.library.cornell.edu/wp/wp-includes/js/jquery/jquery.min.js?ver=3.6.4:2:30158
Deferred/then/l/</t<@https://johnson.library.cornell.edu/wp/wp-includes/js/jquery/jquery.min.js?ver=3.6.4:2:30460
 undefined`

https://stackoverflow.com/questions/980697/element-dispatchevent-is-not-a-function-js-error-caught-in-firebug-of-ff3-0

Seeing this in olinuris also.

**Comment by jgreidy** [comment]
This looks like something to check:

https://stackoverflow.com/questions/72407238/uncaught-typeerror-e-dispatchevent-is-not-a-function-elementor-wordpress-fronte

Two versions of jquery loaded?

**Comment by mhk33** [comment]
I see it called twice on the Mann site! Not sure where it comes from.

view-source:https://mann.library.cornell.edu/staff-profiles

**Comment by jgreidy** [comment]
This is from vue/src/Staff.vue in the culu theme:

`
  mounted () {
    // Import jQuery from Google CDN =(
    // -- dependency for LibCal Consultation (my scheduler) widget
    const jqueryCdn = document.createElement('script')
    jqueryCdn.setAttribute('src', '//ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js')
    document.head.appendChild(jqueryCdn)
  },
`

Not sure if/when it gets used, but the site is running jQuery 3.6.4 and this seems to load 3.4.0.

I got confused by jquery-migrate which is a different thing and has different versions than jquery.

**Comment by jgreidy** [comment]
I made a new upstream and tested this culu JS version fix on OlinUris. No change in the error.

**Comment by sarah-cul** [comment]
Confirming this is still happening, I see both the errors. I can investigate.

**Comment by sarah-cul** [comment]
fixing this in v1.7.1 of the culu theme
